chipsconfig/jms.properties.template

```diff
@@ -4,6 +4,10 @@
 jms.jndiContextFactory=${JMS_JNDICONTEXTFACTORY}
 jms.jndiProviderUrl=t3://${HOSTNAME}:7001
 jms.queueConnectionFactoryName=${JMS_QUEUECONNECTIONFACTORYNAME}
+fes.tdf.allocation=${FES_TDF_ALLOCATION}
```

Comment:
If this line is being moved, shouldn't it go to line 199 to go with the other fes.xxxx property? It's after this section :
```
# Default FES Org Unit ID - should no FES config for the transaction type/reg loc/corporate body type be found, place FES work item in this org unit
fes.default.orgunit.id=${FES_DEFAULT_ORGUNIT_ID}
```

Reply:
I actually hadn't moved this line but the ones before it, the diff just happens to be displayed like this. 
But happy to place these 2 together now that you point it out 👍